tests/domain/test_application_roles.py

```
@@ -33,3 +33,30 @@ def test_enabled_application_role():
     ApplicationRoles.enable(app_role)
 
     assert app_role.status == ApplicationRoleStatus.ACTIVE
+
+
+def test_get():
```

Can we throw in an exception that it raises `NotFoundError` for an invalid ID?